src/panels/lovelace/entity-rows/hui-media-player-entity-row.ts

```diff
@@ -68,47 +94,131 @@ class HuiMediaPlayerEntityRow extends LitElement implements LovelaceRow {
         .config=${this._config}
         .secondaryText=${this._computeMediaTitle(stateObj)}
       >
-        ${OFF_STATES.includes(stateObj.state)
+        ${Boolean(
+          OFF_STATES.includes(stateObj.state)
+            ? supportsFeature(stateObj, SUPPORT_TURN_ON)
+            : supportsFeature(stateObj, SUPPORT_TURN_OFF)
+        )
           ? html`
-              <div class="text-content">
-                ${this.hass!.localize(`state.media_player.${stateObj.state}`) ||
-                  this.hass!.localize(`state.default.${stateObj.state}`) ||
-                  stateObj.state}
-              </div>
+              <paper-icon-button
+                icon="hass:power"
+                @click=${this._togglePower}
+              ></paper-icon-button>
             `
-          : html`
-              <div class="controls">
-                ${stateObj.state !== "playing" &&
-                !supportsFeature(stateObj, SUPPORTS_PLAY)
-                  ? ""
-                  : html`
-                      <paper-icon-button
-                        icon="${this._computeControlIcon(stateObj)}"
-                        @click="${this._playPause}"
-                      ></paper-icon-button>
-                    `}
-                ${supportsFeature(stateObj, SUPPORT_NEXT_TRACK)
-                  ? html`
-                      <paper-icon-button
-                        icon="hass:skip-next"
-                        @click="${this._nextTrack}"
-                      ></paper-icon-button>
-                    `
-                  : ""}
-              </div>
-            `}
+          : ""}
       </hui-generic-entity-row>
+      <div class="flex">
+        <div class="volume">
+          ${supportsFeature(stateObj, SUPPORT_VOLUME_MUTE)
+            ? html`
+                <paper-icon-button
+                  .icon=${stateObj.attributes.is_volume_muted
+                    ? "hass:volume-off"
+                    : "hass:volume-high"}
+                  @click=${this._toggleMute}
+                ></paper-icon-button>
+              `
+            : ""}
+          ${supportsFeature(stateObj, SUPPORT_VOLUME_SET) && !this._narrow
```

Comment:
Let's turn the checks around since boolean checking is faster.
```suggestion
          ${!this._narrow && supportsFeature(stateObj, SUPPORT_VOLUME_SET)
```